_books/ion-1-1/src/whats_new.md

```diff
@@ -224,23 +84,27 @@ Within a list or S-expression, the stream becomes additional child elements in t
 
 **E-expressions in S-expressions**
 ```ion
+// Encoding
 (a (:values 1 2 3) b (:none) c)
+
+// Evaluates to
 (a 1 2 3 b c)
 ```
 
+**E-expressions in structs**
+
 Within a struct at the field name position, the resulting stream must contain structs and each of the fields in those
 structs become fields in the enclosing struct (the value portion is not specified); at the value position, the resulting
-stream of values becomes fields with whatever field name corresponded before the E-expression (empty stream elides the
-field all together). In the following examples, let us define `(:make_struct c 5)` that evaluates to a single struct
+stream of values becomes values with whatever field name corresponded before the E-expression (empty stream elides the
+field all together). In the following examples, let us define `(:make_struct { c: 5 })` that evaluates to a single struct
 `{c: 5}`.
 
-**E-expressions in structs**
 ```ion
 // Encoding
 {
   a: (:values 1 2 3),
   b: 4,
-  (:make_struct c 5),
+  (:make_struct { c: 5 }),
```

Comment:
Should we just replace this with a `make_field` invocation? Or is the idea here to avoid introducing the idea of system macros?

Reply:
I altered this to include both a `make_field` and a `make_struct` invocation, now we're demonstrating both.